<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 213, 213)</string>
  </property>
  <widget class="QLineEdit" name="patientname">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Blood Group:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Date Of Birth:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dob">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Phone Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Address:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BG">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>O+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB-</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="phonenumber">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="address">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="goback">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Patient Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="patientsubmitbutton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
